lib/katello/tasks/repository.rake

```diff
@@ -152,6 +163,10 @@ namespace :katello do
     end
   end
 
+  def handle_missing_root_repo(root_repo)
+    root_repo.destroy!
```

Comment:
Didn't check but I think the roots are being destroyed even when COMMIT is false